workflows/chipseq/chipseq-standard.wdl

```diff
@@ -29,6 +29,7 @@
 
 version 1.0
 
+# TODO should we enfore an order to imports? Alphabetical? Order referenced? URLs before/after relatives?
```

Comment:
I think we should enforce an order. But I'm pretty ambivalent towards what it is. Probably alphabetical is the right approach? Do we take into special consideration paths/URLs?

Reply:
I think alphabetical is fine. Internal before external? Add a separator between them? I would say yes to both.

Reply:
> Internal before external? Add a separator between them?

No opposition to either of these. Let's adopt those rules.

But I'm still unsure how to handle paths (in either relative or URL imports). e.g. does `./rnaseq-core.wdl` come before or after `../../tools/util.wdl`? Are we sorting on `basename`? How do we handle the case of `../foo.wdl` and `../../bar/foo.wdl`?

I think the easiest solution is just do a naive sort on the strings. No special handling for paths/basenames. As long as it's consistent I'll be happy, so I vote for whatever order is easiest to implement. There are smarter sorts out there but I don't think we want/need them.

Reply:
I would generally sort on basename, but maybe that's not a good approach. I think most of the grouping that you'd want would be handled by internal vs. external. However, it may be desirable to sort the full path, so that everything under `tools/` would be together and separate from something like `structs/`.

For the case of `../foo.wdl` and `../../bar/foo.wdl`, you'd have to rename them internally with an `as` correct? If so I'd sort them with the plain import then the `as` disambiguated one (or on the `as` values if both have them).